web/src/app/page.tsx

```diff
@@ -54,9 +62,11 @@ export default async function Home() {
 	const stars = await getGitHubStars()
 
 	return (
-		<div className="flex flex-col min-h-screen">
-			<HeroSection totalIcons={totalIcons} stars={stars} />
-			<RecentlyAddedIcons icons={recentIcons} />
-		</div>
+		<>
+			<div className="flex flex-col min-h-screen">
+				<HeroSection totalIcons={totalIcons} stars={stars} />
+				<RecentlyAddedIcons icons={recentIcons} />
+			</div>
+		</>
```

Comment:
not necessary since <> and </> literally don't add any code to the page

Reply:
Linter added this lol